src/rules.js

```diff
@@ -0,0 +1,335 @@
+import propertiesOrder from "./declaration-block-properties-order"
+
+/**
+ * Rules settings
+ *
+ * @see http://stylelint.io/user-guide/rules/
+ *
+ * @type Object
+ */
+const rules = {
+  //
+  // Color
+  // ====================
+
+  "color-hex-case": "lower", // Specify lowercase or uppercase for hex colors.
+  "color-hex-length": "short", // Specify short or long notation for hex colors.
+  "color-named": "never", // Require (where possible) or disallow named colors.
+  // "color-no-hex": true, // Disallow hex colors.
```

Comment:
Are you sure?
I like the possibility to see all existing rules and activate the ones I want. :) WYT?